macros/sql/get_column_values.sql

```diff
@@ -36,16 +37,26 @@ Returns:
 
         {%- else -%}
 
-            select
-                {{ column }} as value
-
-            from {{ target_relation }}
-            group by 1
-            order by count(*) desc
-
-            {% if max_records is not none %}
-            limit {{ max_records }}
-            {% endif %}
+            with sorted_column_values as (
+
+                select
+                    {{ column }} as value,
+                    {%- if sort_column is none %}
+                    count(*) as sort_column
+                    {% else %}
+                    {# We take the max sort value for each value to make sure 
+                        there are no duplicate rows for each value #}
+                    max({{ sort_column }}) as sort_column
```

Comment:
huh, this is interesting. I thought this PR would be to make it easier to alpha-sort, i.e. "order by {{ column }}" (which I typically achieve through a Jinja sort)

What kind of ways do you want to sort these results?

Reply:
This is intended to let you sort by **frequency** in **ascending** (or descending) order, which I think was was not possible previously and is not possible in Jinja with just the results since the counts aren't returned. 

(This is based on an internal version of this we had to implement at Quibi as a result.)